main.star

```diff
@@ -227,6 +228,7 @@ def launch_artifact_server(plan, uploaded_files):
     )
 
 
+# Will question: when does this happen compared to everything else? Do we generate the files before we generate the final starlark config or is it evaluated after?
```

Comment:
```suggestion
```

Reply:
This is the first thing that we do in line 17 in the `main()` function; for now the `generated` files are files from `optimism/.devnet/` generated via `make devnet-allocs` we can dynamically generate them